tests/generic/test_accepted_values_by_relation.sql

```diff
@@ -0,0 +1,31 @@
+{% test accepted_values_by_relation(model, column_name, to, field, quote=True) %}
+
+    {% set accepted_values = dbt_utils.get_column_values(
```

Comment:
Since [get_column_values](https://github.com/dbt-labs/dbt-utils/blob/main/macros/sql/get_column_values.sql) takes a couple more arguments one could go even more fancy here. 

defining `default` for example, allows the macro to behave like the existing `accepted_values` test.